### ISSUE TITLE
[Housekeeping] Refactor typography, sizes etc.

### ISSUE THREAD

**Issue by jkaltoft** [kirbydesign/designsystem#2313]
#2301 has revealed that we have implemented font-sizes etc. in an unfortunate way.

Trying to control sizes in pixel units is doomed to fail. First and foremost, it prevents users from controlling the base font-size which can result in very poor accessibility. Our ideal font-sizes may not be readable by all. See e.g. https://developer.mozilla.org/en-US/docs/Web/CSS/length#absolute_length_units

This issue proposes a complete refactorization of sizes so everything is sized relative to a given base font-size. Making sizes relative is not a complex task, but I'm not certain how comprehensive this will be, so we should start out by creating a PoC to get an idea of the extent of it.

If Kirby is ever going to be "web/desktop ready" and accessible, then this rewrite is necessary.

**Comment by RasmusKjeldgaard** [comment]
Closed as duplicate of issues mentioned here: #3586
